src/commands/readme.ts

```diff
@@ -32,15 +32,18 @@ Customize the code URL prefix by setting oclif.repositoryPrefix in package.json.
 
   async run() {
     const {flags} = this.parse(Readme)
-    const config = await Config.load({root: process.cwd(), devPlugins: false, userPlugins: false})
+    const cwd = process.cwd()
+    const readmePath = path.resolve(cwd, 'README.md')
+    const config = await Config.load({root: cwd, devPlugins: false, userPlugins: false})
+
     try {
-      const p = require.resolve('@oclif/plugin-legacy', {paths: [process.cwd()]})
+      const p = require.resolve('@oclif/plugin-legacy', {paths: [cwd]})
       const plugin = new Config.Plugin({root: p, type: 'core'})
       await plugin.load()
       config.plugins.push(plugin)
     } catch {}
-    await config.runHook('init', {id: 'readme', argv: this.argv})
-    let readme = await fs.readFile('README.md', 'utf8')
+    await (config.runHook as any)('init', {id: 'readme', argv: this.argv})
```

Comment:
Why the need for the `as any`?

Reply:
Not sure, will investigate.

Reply:
Ah yes, it’s due to this: https://github.com/oclif/config/pull/158

Reply:
👍